lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -615,4 +633,16 @@ export abstract class BaseQueryBuilderService {
             queryParamsHandling: 'merge'
         });
     }
+
+    private readonly queryFragmentsHandler: ProxyHandler<{ [key: string]: any }> = {
```

Comment:
queryFragmentsHandler is private readonly field. It's not a function. So please move it above class to place where all fields are placed